tests/test_integrated_channels/test_cornerstone/test_exporters/test_content_metadata.py

```diff
@@ -31,6 +31,14 @@ def setUp(self):
         self.config = factories.CornerstoneEnterpriseCustomerConfigurationFactory(
             enterprise_customer=self.enterprise_customer_catalog.enterprise_customer,
         )
+        self.global_config = factories.CornerstoneGlobalConfigurationFactory(
```

Comment:
I think we should store subject mapping like this to voice duplicate values and to improve readability of mapping
```
subject_mapping={
                "Technology": ["Computer Science"],
                "Business Skills": ["Communication"],
                "Creative": ["Music", "Design"],
            }
```